<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QRadioButton" name="radioButton_wuxian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>108</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>无限制</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>48</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>默认数量(10)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设置系统最大设备数：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_renyi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>78</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>78</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32766</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
